src/templates/stickyboard-simple/src/components/base/PageBase.js

```diff
@@ -132,6 +150,8 @@ class PageBase extends React.Component {
                         ) {
                             return (
                                 <Sticker key={block.i}>
+                                    {this.state.deleteButtonToggle ? 
+                                    <DeleteIcon onClick = {() => this.handleDelete(block.i)} />  : ""}
```

Comment:
You can use **deleteButtonToggle** instead of **this.state.deleteButtonToggle**, because it is already declared in the line 130.

Refer to the link below(Object destructuring section)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

And if you want to show or hide a component depends on the variable,
Please use && operator.
For example,
`{showComponent && <Component />}`